source/usage-examples/findOne.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+func main() {
+
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	// Disconnect the client once the function returns.
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+	coll := client.Database("sample_mflix").Collection("movies")
+	var result bson.D
+	opts := options.FindOne()
```

Comment:
The option functions can be chained. This can be simplified to:

```
opts := options.FindOne().
    SetProjection(bson.D{{"_id", 0}, {"title", 1}, {"imdb", 1}}).
    SetSort(bson.D{{"imdb.rating", -1}})
```

Reply:
Thank you for the note. For some reason I thought breaking them out in to lines made them more readable, but seeing your suggested change I agree that this improves the code snippet. Thanks for pointing this out.

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/efdd69ce4f9705bbdfb9c74ef7875e2521fe0169